pages/Gyms/main.html

```diff
@@ -78,6 +80,8 @@ <h3>Pokémon Team</h3>
                         <a class="text-decoration-none" href="#!" data-bind="text: $data.name, attr: {href: `#!Pokemon/${$data.name}` }"></a>
                     </td>
                     <td class="align-middle" data-bind="text: $data.maxHealth.toLocaleString()"></td>
+                    <td class="align-middle" data-bind="text: $data.level.toLocaleString()"></td>
+                    <td class="align-middle" data-bind="text: (pokemonList.find(p => p.name == $data.name).exp * 1.5).toLocaleString()"></td>
```

Comment:
```suggestion
                    <td class="align-middle" data-bind="text: (pokemonMap[$data.name].exp * 1.5).toLocaleString()"></td>
```
You can just use `pokemonMap` here and the other 2 places

Reply:
Done